letsencrypt-auto-source/pieces/bootstrappers/deb_common.sh

```diff
@@ -21,7 +21,7 @@ BootstrapDebCommon() {
     QUIET_FLAG='-qq'
   fi
 
-  $SUDO apt-get $QUIET_FLAG update || echo apt-get update hit problems but continuing anyway...
+  $SUDO apt-get $QUIET_FLAG update || say apt-get update hit problems but continuing anyway...
```

Comment:
I think this should be `error` so the user is notified that we encountered problems.